topics/proteomics/tutorials/ml-modeling-of-anti-cancer-peptides/tutorial.md

```diff
@@ -0,0 +1,284 @@
+>
+>
+{: .hands_on}
+
+
+
+## Background
+
+Biological molecules such as proteins, peptides, DNA, and RNA can be represented by their biochemical or sequences-based properties. These properties can be utilized to deduce biological meanings. Properties associated with a peptide sequence such as overall charge, hydrophobicity profile, or k-mer composition can be utilized to build a machine learning model and predict the biological properties of unknown peptides. Finding anticancer peptides (ACPs) through wet-lab methods is costly and time-consuming; thus, the development of an efficient computational approach is useful to predict potential ACP peptides before wet-lab experimentation.
+In this tutorial, we used 6 different machine learning algorithms for the prediction of ACPs using the peptide sequence-based CTD (Composition transition and distribution descriptors) descriptors or features. All the models were trained using our dataset that combines 138 Anticancer peptides and 138 non-anticancer peptides. We applied 10 fold cross-validation on this data set.
+
+
+## Calculating Peptide Descriptors 
+
+A descriptor or feature is the quantitative or a qualitative measure of a property that is associated with a sequence. For example, a chemical compound can be described via its charge chemical formula, molecular weight, number of rotatable bonds, etc. Similarly, several properties can be deduced from the biological sequence that can be utilized to summarise a biological property such as anti-cancer activity. In this example, we utilized CTD descriptors, describes as composition, transition, and distribution descriptors. 
+
+![Alternative text](../../images/PDAUG_ML_1.png "ML algorithms use numerical representation of a sequence-based properties for model building")
+
+### Calculating descriptor for ACPs
+
+> ### {% icon hands_on %} Hands-on: Task description
+>
+> 1. **PDAUG Sequence Property Based Descriptors** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Input fasta file"*: `ACPs.fasta` (output of **Input dataset** {% icon tool %})
+>    - *"DesType"*: `CTD`
+>
+>
+{: .hands_on}
+
+### Calculating descriptor for non-ACPs
+
+> ### {% icon hands_on %} Hands-on: Task description
+>
+> 1. **PDAUG Sequence Property Based Descriptors** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Input fasta file"*: `non_ACPs.fasta` (output of **Input dataset** {% icon tool %})
+>    - *"DesType"*: `CTD`
+>
+>
+{: .hands_on}
+
+
+## Preparing a traning data set
+
+We will combine the ACPs and non-ACPs data set as a single data frame and will add the class label. 
+
+### Adding class label in training data
+
+
+The class label usually describes samples from two different groups, in our case ACPs and non-ACPs.   In the case of binary classification, usually, samples are labeled as "0" or 1. 
+
+
+- **We marked ACPs as "1" or positive data.** 
+
+> ### {% icon hands_on %} Hands-on: Task description
```

Comment:
These hands on boxes could use a description of the task. Additionally they should probably be combined.

Reply:
Description added